shared/src/common/enums/user/user.ts

```diff
@@ -1 +1,2 @@
 export { UserValidationMessage } from './user-validation-message.enum';
+export { UserValidationRule } from '../validation/validation.enum';
```

Comment:
```suggestion
export { UserValidationRule } from '../validation/validation';
```
this import should be in `shared/src/common/enums`

everything should be flat